web/book/src/reference/syntax/README.md

```diff
@@ -8,24 +8,27 @@ A summary of PRQL syntax:
-| `` ` ` ``       | [Quoted identifiers](./keywords.md#quoting)                                    | ``select `first name` ``                                |
-| `#`             | [Comments](./comments.md)                                                      | `# A comment`                                           |
-| `@`             | [Dates & times](./literals.md#date-and-time)                                   | `@2021-01-01`                                           |
-| `==`            | [Self-equality in `join`](../stdlib/transforms/join.md#self-equality-operator) | `join s=salaries (==id)`                                |
-| `->`            | [Function definitions](../declarations/functions.md)                           | `let add = a b -> a + b`                                |
-| `=>`            | [Case statement](./case.md)                                                    | `case {a==1 => c, a==2 => d }`                          |
-| `+`/`-`         | [Sort order](../stdlib/transforms/sort.md)                                     | `sort {-amount, +date}`                                 |
-| `??`            | [Coalesce](./operators.md#coalesce)                                            | `amount ?? 0`                                           |
+| Syntax                 | Usage                                                                          | Example                                                 |
+| ---------------------- | ------------------------------------------------------------------------------ | ------------------------------------------------------- |
+| <code>\|</code>        | [Pipelines](./function-calls.md)                                               | <code>from employees \| select first_name</code>        |
+| `=`                    | [Assigns](./tuples.md)                                                         | `from e = employees` <br> `derive total = (sum salary)` |
+| `:`                    | [Named args & parameters](../declarations/functions.md)                        | `interp low:0 1600 sat_score`                           |
+| `{}`                   | [Tuples](./tuples.md)                                                          | `{id, false, total = 3}`                                |
+| `[]`                   | [Arrays](./arrays.md)                                                          | `[1, 4, 3, 4]`                                          |
+| `! && == +`            | [Operations](./operators.md)                                                   | <code>filter a == b + c \|\| d >= e</code>              |
+| `()`                   | [Parentheses](./operators.md#parentheses)                                      | `derive celsius = (fahrenheit - 32) / 1.8`              |
+| `123`,`5e10`,`100_000` | [Numbers](./literals.md#numbers)                                               | `derive { huge = 5e10 }`                                |
```

Comment:
I think this is the main addition, good idea

Reply:
The major change is inclusion of the remaining items in the Literals page: Numbers, Boolean, Null.